--- bazel/BUILD
+++ bazel/BUILD
@@ -44,6 +44,9 @@
     ],
     configure_env_vars = {
         "AR": "ar_wrapper",
+        "CFLAGS":"-fno-sanitize=all",
+        "CXXFLAGS":"-fno-sanitize=all",
+        "LDFLAGS": "-stdlib=libc++ -B lld -lc++ --rtlib=compiler-rt --unwindlib=libunwind -fno-sanitize=all",
     },
     configure_options = [
         "--disable-nls",
@@ -68,6 +71,9 @@
         "M4": "$$EXT_BUILD_DEPS$$/m4/bin/m4",
         "CC_FOR_BUILD": "$$CC$$",
         "AR": "ar_wrapper",
+       "CFLAGS":"-fno-sanitize=all",
+        "CXXFLAGS":"-fno-sanitize=all",
+        "LDFLAGS": "-stdlib=libc++ -B lld -lc++ --rtlib=compiler-rt --unwindlib=libunwind -fno-sanitize=all",
     },
     lib_source = "@bison//:all",
     static_libraries = ["liby.a"],
@@ -92,6 +98,9 @@
     configure_env_vars = {
         "M4": "$$EXT_BUILD_DEPS$$/m4/bin/m4",
         "AR": "ar_wrapper",
+       "CFLAGS":"-fno-sanitize=all",
+        "CXXFLAGS":"-fno-sanitize=all",
+        "LDFLAGS": "-stdlib=libc++ -B lld -lc++ --rtlib=compiler-rt --unwindlib=libunwind -fno-sanitize=all",
     },
     lib_source = "@flex//:all",
     tools_deps = [":ar_wrapper"],
